packages/web/lib/page.js

```diff
@@ -24,16 +24,25 @@ const getNumber = (el) => {
 };
 const getUrl = (el) => el.href || null;
 
-/**
- * A scope accessible within a `page` RuleSet
- */
 const getPageScope = (parent) => {
+  /**
+   * @scope { A scope accessible within a `page` RuleSet }
+   */
   const pageScope = {
     parent,
     vars: {},
     internal: {
       page: null,
-      config: {},
+      /**
+       * @config {
```

Comment:
`config` is just special to this page context right?

Or did you want to make this a general pattern for all browse scopes to support a `config` rule, and then the docs support `@config` too?

Reply:
Yea, I assumed that @config would be a general rule for any scope since we have a page config, and we also wanted a browse config for browser configuration